lib/active_record_doctor/detectors/incorrect_length_validation.rb

```diff
@@ -33,28 +33,47 @@ def message(model:, attribute:, table:, database_maximum:, model_maximum:)
       def detect
         each_model(except: config(:ignore_models), existing_tables_only: true) do |model|
           each_attribute(model, except: config(:ignore_attributes), type: [:string, :text]) do |column|
-            model_maximum = maximum_allowed_by_validations(model, column.name.to_sym)
-            next if model_maximum == column.limit
+            model_maximum = maximum_allowed_by_length_validation(model, column.name.to_sym)
+            database_maximum = column.limit
+            next if model_maximum == database_maximum
+            next if column.limit && covered_by_inclusion_validation?(model, column.name.to_sym, database_maximum)
```

Comment:
Since we set `database_maximum` to `column.limit`, I suggest we don't reference `column.limit` here but use `database_maximum` consistently.